src/logic_analyzer.cpp

```diff
@@ -311,20 +317,21 @@ LogicAnalyzer::LogicAnalyzer(struct iio_context *ctx,
 	trigger_settings_ui->setupUi(trigger_settings);
 	ui->triggerSettingsLayout->insertWidget(0, trigger_settings);
 	setupTriggerSettingsUI();
-	QAbstractItemView *trigVw = trigger_settings_ui->cmb_trigg_extern_cond_1->view();
-	QListView* listVw = qobject_cast<QListView*>(trigVw);
-	listVw->setSpacing(2);
-	trigVw = trigger_settings_ui->cmb_trigg_extern_cond_2->view();
-	listVw = qobject_cast<QListView*>(trigVw);
-	listVw->setSpacing(2);
+
+	trigger_settings_ui->extern_to->setVisible(false);
 
 	/* Cursor Settings */
 	connect(trigger_settings_ui->cmb_trigg_extern_cond_1,
 		SIGNAL(currentIndexChanged(int)),
 		this, SLOT(setExternalTrigger(int)));
-	connect(trigger_settings_ui->cmb_trigg_extern_cond_2,
+
+	for(auto val : externalTriggerSourceMap) {
```

Comment:
The items in the `externalTriggerSourceMap` could be accessed by constant reference : `const auto &val`

Reply:
fixed